### PR TITLE
HTML Kitchen Sink Update

### PR DESCRIPTION
**Pull request by bobwise**
* Upgrade to Spark v 15
* Add Rob's webpack and eslint changes
* Add navigation to the site
* Split components into individual pages
* Add functional examples for spinner button

TODO after the release or when the final Spark version is known, update package.json and package-lock.json.

### CODE REVIEW

File: src/styles/navigation.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  padding: $sprk-space-inset-short-s;
+  padding-top: $sprk-space-s;
+  padding-bottom: $sprk-space-s;
+  margin-bottom: $sprk-space-s;
+  border: 0;
+  border-top: $sprk-tab-navigation-btn-border-bottom;
+  color: $sprk-black;
+}
+
+.#{$app-namespace}-c-AngularNav a:hover,
```
Comment (afebbraro) [inline src/styles/navigation.scss:46]: Lets update to not be "Angular".

**Review by afebbraro — CHANGES_REQUESTED** [review]
Let a comment about angular namespacing

**Review by afebbraro — APPROVED** [review]
Looks good, we'll want to wait to merge because of the pkg bump.

**Comment by bobwise** [comment]
duplicate PR

**Pull request closed without merging**